docs/Operators.md

```diff
@@ -1091,7 +1091,7 @@ opset_import {
 #### Attributes
 
 <dl>
-<dt><tt>alpha</tt> : float</dt>
+<dt><tt>alpha</tt> : float (required)</dt>
```

Comment:
This should be a optional attribute, and default value is 1.0.

Reply:
Good catch! Fixed.